golem_messages/message/base.py

```diff
@@ -1,723 +1,728 @@
-        'challenge',
-        'difficulty',
-        'metadata',
-        '_version',
-    ] + Message.__slots__
-
-    @classmethod
-    def deserialize_with_header(cls, header, data, *args, **kwargs):  # noqa pylint: disable=arguments-differ
-        raw_version = data[-cls.VERSION_LENGTH:]
-        data = data[:-cls.VERSION_LENGTH]
-        try:
-            str_version = struct.unpack(cls.VERSION_FORMAT, raw_version)[0] \
-                .decode('ascii', 'replace')
-        except struct.error as e:
-            raise exceptions.VersionMismatchError(
-                "Unreadable version {raw_version}".format(
-                    raw_version=raw_version,
-                )
-            ) from e
-        verify_version(str_version)
-        instance = super().deserialize_with_header(
-            header,
-            data,
-            _version=str_version,
-            *args,
-            **kwargs,
-        )
-        return instance
-
-    def serialize(self, *args, **kwargs):  # pylint: disable=arguments-differ
-        serialized = super().serialize(*args, **kwargs)
-        version = struct.pack(
-            self.VERSION_FORMAT,
-            self._version.encode('ascii', 'replace')
-        )
-        return serialized + version
-
-    def get_short_hash(self, *args, **kwargs):  # noqa pylint: disable=arguments-differ
-        return super().get_short_hash(*args, **kwargs) \
-            + self._version.encode('ascii', 'replace')
-
-    def __init__(self, **kwargs):
-        super().__init__(**kwargs)
-        deserialized = kwargs.pop('deserialized', False)
-        if not deserialized and not hasattr(self, '_version'):
-            self._version = golem_messages.__version__
-
-    def __repr__(self):
-        return "<{} _version:{}>".format(
-            super().__repr__(),
-            getattr(self, '_version', '<undefined>'),
-        )
-
-    def __eq__(self, obj):
-        if not self._version == getattr(obj, '_version', None):
-            return False
-        return super().__eq__(obj)
-
-
-@library.register(1)
-class RandVal(Message):
-    """Message with signed random value"""
-    __slots__ = ['rand_val'] + Message.__slots__
-
-
-@library.register(2)
-class Disconnect(AbstractReasonMessage):
-    ENCRYPT = False
-
-    __slots__ = AbstractReasonMessage.__slots__
-
-    class REASON(enum.Enum):
-        DuplicatePeers = 'duplicate_peers'
-        TooManyPeers = 'too_many_peers'
-        Refresh = 'refresh'
-        Unverified = 'unverified'
-        ProtocolVersion = 'protocol_version'
-        BadProtocol = 'bad_protocol'
-        Timeout = 'timeout'
-        NoMoreMessages = 'no_more_messages'
-        WrongEncryption = 'wrong_encryption'
-        ResourceHandshakeFailure = 'resource_handshake'
-        KeyNotDifficult = 'key_not_difficult'
-        Bootstrap = 'bootstrap'
-
-
-@library.register(3)
-class ChallengeSolution(Message):
-    __slots__ = ['solution'] + Message.__slots__
+import calendar
```

Comment:
![image](https://user-images.githubusercontent.com/3904252/50649315-d6c3de00-0f7d-11e9-99e8-4d65169b2b98.png)

Reply:
CRLF -> LF and I didn't notice... :(

Reply:
If you're changing back to the correct line endings please do it in separate PR as it's awkward to see what has been changed in this file